controllers/main.go

```diff
@@ -1848,7 +1848,14 @@ type TicketInfoInvalid struct {
 	Ticket string
 }
 
-// TicketInfoLive represents live or immature (mined) tickets that have yet to
+// TicketInfoImmature represents immature (mined) tickets that have yet to
+// be spent by either a vote or revocation.
+type TicketInfoImmature struct {
```

Comment:
Im not very keen on the duplication here. We already have a struct below named `TicketInfoLive` and it has the same fields as this new struct.

Perhaps we could rename the existing one something more generic, and reuse that one.